docs/chapters/incremental_indexing/index.md

```diff
@@ -0,0 +1,27 @@
+## Generating Documents Ids:
+
+In Jina, the document ID is by default generated deterministically by computing a hash the document content. This avoids repeating the doc id while reindexing by using a Random or Incremental counter. The user can `override` document ids with Jina client and add `tags` to map to their unique concepts.
+
+## Incremental Indexing Design Pattern:
```

Comment:
I would avoid calling it design pattern, since there is a different section on `common design patterns`